lib/graphQl/index.js

```diff
@@ -13,10 +13,12 @@ const writeTypes = require('./writeTypes.js')
 jsonApiGraphQL.with = app => {
   const config = jsonApi._apiConfig
 
-  app.use(new RegExp(`${config.base}$`), graphqlHTTP({
-    schema: jsonApiGraphQL.generate(jsonApi._resources),
-    graphiql: !!config.graphiql
-  }))
+  if (config.graphiql !== false) {
```

Comment:
Shall we introduce the `strict` configuration property that @theninj4 [suggests](https://github.com/holidayextras/jsonapi-server/pull/233#issuecomment-269498049) to turn off the GraphQL endpoint? Abusing the `graphiql` configuration property for this end is not the best option.